### DISCUSSION TITLE
When using 10 minute inspection cheats, Crooked House should be set to "Never" inspect

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#21057] [Feature requests]
The Crooked House does not break down, therefore it is not an optimization to automatically set the inspection time to 10 minutes. I believe we should change the default behavior so when 10 minute inspections is enabled in the options, the optimized value of "Never inspected" is used for the Crooked House. The point of the option is for more reliable rides, but what is currently happening is mechanics are wasting their time inspecting Crooked Houses.

**Comment by ocalhoun6** [comment]
Should also apply to the other ride types which cannot break down.

**Reply by 733737** [reply]
do you know what other ones there are?

**Reply by ocalhoun6** [reply]
- Other reskins of the crooked house
- Boat hire (only with certain vehicles, so maybe don't change the inspection intervals on that)
- Maze
- Lift?
- Mini-Golf?

There may be others I'm not thinking of at the moment.

**Reply by ocalhoun6** [reply]
On second thought, doesn't inspection interval have something to do with the "safest park in the country" award? Having some rides never inspected might mess with that?

**Reply by 733737** [reply]
I don't think awards are more important than proper park management. And if you are correct we need to change that award logic as well.
